simuvex/procedures/syscalls/mmap.py

```diff
@@ -82,10 +82,6 @@ def run(self, addr, length, prot, flags, fd, offset): #pylint:disable=arguments-
             # Can't give you that address. Find a different one.
             addr = self.allocate_memory(size)
```

Comment:
Line 52 should be fine, but I see the problem with line 83, this should actually happen in a loop, otherwise the address returned might not be mapped.